tests/Endpoints/AbstractEndpoint.php

```diff
@@ -236,21 +253,22 @@ protected function createGraphQLRequest(
 
     /**
      * Test getting a single value from the API
+     * @throws Exception
```

Comment:
After looking at a bunch more of them I'm really disliking how hard they make it to review these changes. I'd also drop params in docs where they are declared explicitly. Can be a future refactor if I can figure out a way to prevent these with linting.

Reply:
i'm not too fond of these either. they were just really easy to add from the "problems" pane in the IDE, so i mindlessly click-added them while in there.

removed.

:heavy_check_mark: